examples/chapter1/CMakeLists.txt

```diff
@@ -83,6 +83,17 @@ TARGET_INCLUDE_DIRECTORIES(${PROJ_NAME}_4b
   ${CMAKE_CURRENT_SOURCE_DIR}/include 
   )
 
+ADD_EXECUTABLE(${PROJ_NAME}_4c ${${PROJ_NAME}_HEADERS} src/example4c.cpp )
```

Comment:
maybe rename the cpp and target to be `eventloop_benchmark_example`[.cpp]

Reply:
Done